docs/admin-manual/file-and-cred-transfer.rst

```diff
@@ -420,29 +420,35 @@ of the checkpoint.
 Enabling the Fetching and Use of OAuth2 Credentials
 ---------------------------------------------------
 
-HTCondor supports two distinct methods for using OAuth2 credentials.
-One uses its own native OAuth client and credential monitor, and one uses
-a separate Hashicorp Vault server as the OAuth client and secure refresh
-token storage.  Each method uses a separate credmon implementation and rpm
-and have their own advantages and disadvantages.
+HTCondor supports multiple methods for using OAuth2 credentials:
+1. Using an OAuth2 client running on webserver local to the access point and HTCondor's OAuth2 credential monitor,
+2. using HTCondor's own local SciToken credential issuer and credential monitor, and/or
+3. using a separate Hashicorp Vault server as the OAuth client and secure refresh
+token storage and HTCondor's Vault credential monitor.
```

Comment:
The phrase "local webserver OAuth2 client" makes my head hurt, and I theoretically understand what's going on.  Is "local webserver" important, or just an implementation detail, compared to something like the following?  [GitHub won't let me use its suggestion format across blank lines for some reason, so I can't fix lines 420 or 429 in the same diff.]

```suggestion
Allowing Jobs to Use OAuth2 Credentials
------------------------------------------------------

HTCondor can automatically renew OAauth2 credentials once obtained and forward short-lived "use" tokens to jobs so they can access OAuth2-protected resources (during input file transfer, job execution, and output file transfer); HTCondor will update the job's copy of "use" tokens so that they don't expire.

A submitter can tell HTCondor about OAuth tokens they obtained using the `htcondor credential add` [FIXME: link] command, but in most cases, HTCondor will obtain tokens on the submitter's behalf based on the jobs they're submitting.  HTCondor submits multiple methods for doing so:

1.  Obtain tokens from an existing OAuth issuer.  When submitting a job, HTCondor will print a URL for the submitter to authenticate HTCondor's request for the necessary tokens.  This is sometimes called "the native OAuth client."
2.  If the AP is trusted by the OAuth2-protected resources, HTCondor can simply issue tokens on its behalf.  This approach is known as "local issuer," because no external service is involved.  No submitter involvement is required.
3.  Via a Hashicorp Vault [FIXME: link] service, which functions as a proxy in this case.  If the submitter has a Kerberos ticket, the OAuth2 tokens may be obtained without submitter involvement.

[FIXME: reorder the subsequent detail paragraphs to match the order of the list above, or vice-versa.]
```

Reply:
From the user's POV, the `ClientCredmon` is equivalent to the `LocalCredmon`, so I guess when that gets documented, it would fit nicely in point 2?